cmake/Modules/CodeCoverage.cmake

```diff
@@ -192,6 +192,10 @@ function(ADD_CODE_COVERAGE)
         COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_BINARY_DIR}/python_base_coverage/${Coverage_NAME}_base_python.xml ${PROJECT_BINARY_DIR}/ || echo "WARNING: No base python xml to copy"
         DEPENDS create_python_base_coverage_dir
                 ${_code_coverage_SOURCE_DIR}/scripts/generate_base_coverage.py
+                ${PROJECT_SOURCE_DIR}/bin
+                ${PROJECT_SOURCE_DIR}/node_scripts
```

Comment:
What does happen if these directory does not exist? If it works, please resolve this comment.

Reply:
That's right...
Hmm... I will check and list up existing directories.

Reply:
If so, I think listing up existing directories should depend on `${PROJECT_SOURCE_DIR}`.  
If a directory is added, the script should run again. So difficult.

Reply:
I solved the issue in https://github.com/GITAI/code_coverage/pull/6/commits/3ce65d32910ab981aeb76af2072e43f30e81cd61

Reply:
@knorth55 Does cmake run again if `bin` directory is newly added with this implementation?

Reply:
Yes, I think so.
I tested with `turtle_tf2`.

### Before adding `bin`

```
catkin b turtle_tf2 && catkin build turtle_tf2 --no-deps --catkin-make-args turtle_tf2_coverage_report
```

```
$ ls turtle_tf2_coverage_report_base_python.xml 
ls: cannot access 'turtle_tf2_coverage_report_base_python.xml': No such file or directory
```

### After adding `bin`

```
roscd turtle_tf2
ln -sf nodes bin
```

```
catkin b turtle_tf2 && catkin build turtle_tf2 --no-deps --catkin-make-args turtle_tf2_coverage_report
```

```
$ ls turtle_tf2_coverage_report_base_python.xml 
turtle_tf2_coverage_report_base_python.xml
```

Reply:
Oh, I didn't know that. If it reruns, it's fine. Thanks.

Reply:
it seems it's rerunning
I think it is because the coverage depends on `_run_tests_${PROJECT_NAME}` target.
`_run_tests_${PROJECT_NAME}` first clean up test results and run the test again, so it will rerun the script everytime.